tasks/centos.yml

```diff
@@ -0,0 +1,13 @@
+- name: "Define proper service name"
```

Comment:
Is this ok ? I mean, can't we specify variables according to the distribution like we do for the tasks ?